apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -27,7 +28,24 @@ function SectionProgressSelector({
   sectionId,
   hasSeenProgressTableInvite,
 }) {
-  const [hasJustToggledViews, setHasJustToggledViews] = React.useState(false);
+  const [hasJustToggledViews, setHasJustToggledViews] = useState(false);
+
+  useEffect(() => {
+    const params = queryParams('view');
+    if (params === 'v2') {
+      setShowProgressTableV2(true);
+      setHasJustToggledViews(true);
+    }
+  }, [setShowProgressTableV2, setHasJustToggledViews]);
+
+  const removeQueryParams = () => {
+    const url =
+      window.location.protocol +
+      '//' +
+      window.location.host +
+      window.location.pathname;
+    window.history.pushState({path: url}, '', url);
```

Comment:
```suggestion
      '${window.location.protocol}//
      ${window.location.host}${window.location.pathname}';
    window.history.pushState({path: url}, '', url);
```
nit: I've been preferring this kind of string concatenation because I think it makes it clearer what the end result will look like. Totally optional if you disagree though.

Reply:
This one was rougher for me to like :)

If it is optional to you, I think I will stick with the original here.